alerts/resource_alerts.libsonnet

```
@@ -36,18 +36,34 @@ local utils = import '../lib/utils.libsonnet';
           } +
           if $._config.showMultiCluster then {
             expr: |||
-              sum(namespace_cpu:kube_pod_container_resource_requests:sum{%(ignoringOverprovisionedWorkloadSelector)s}) by (%(clusterLabel)s) - (sum(kube_node_status_allocatable{%(kubeStateMetricsSelector)s,resource="cpu"}) by (%(clusterLabel)s) - max(kube_node_status_allocatable{%(kubeStateMetricsSelector)s,resource="cpu"}) by (%(clusterLabel)s)) > 0
+              (sum(namespace_cpu:kube_pod_container_resource_requests:sum{%(ignoringOverprovisionedWorkloadSelector)s}) by (%(clusterLabel)s) -
+              sum(kube_node_status_allocatable{%(kubeStateMetricsSelector)s,resource="cpu"}) by (%(clusterLabel)s) > 0
               and
-              (sum(kube_node_status_allocatable{%(kubeStateMetricsSelector)s,resource="cpu"}) by (%(clusterLabel)s) - max(kube_node_status_allocatable{%(kubeStateMetricsSelector)s,resource="cpu"}) by (%(clusterLabel)s)) > 0
+              count by (cluster) (max by (cluster, node) (kube_node_role{role="control-plane"})) < 3)
```

lgtm! Couple of points for approval:

1. `%(clusterLabel)s` to be used instead of `cluster`
2. `%(kubeStateMetricsSelector)s` to be used for `kube_node_role`

I've included an example below but there are a few more instances to change in this PR:

```suggestion
              count by (%(clusterLabel)s) (max by (%(clusterLabel)s, node) (kube_node_role{%(kubeStateMetricsSelector)s, role="control-plane"})) < 3)
```

---

Ah, right! I've amended the patch. 👍🏼

---

Tests need patching, on it.